scripts/sandbox.ts

```diff
@@ -197,7 +207,10 @@ async function addStories(paths: string[], { mainConfig }: { mainConfig: ConfigF
 async function main() {
   const optionValues = await getOptions();
 
-  const { template, forceDelete, forceReuse, link, dryRun } = optionValues;
+  const { template, forceDelete, forceReuse, link, dryRun, debug } = optionValues;
+
+  await ensureDir(sandboxDir);
```

Comment:
I'm great with doing this but is it not the case that `yarn repro-next` silently fails if the containing directory doesn't exist? (That's what I saw in my testing). I think we ought to fix that (also?)

Reply:
The problem was that the previous implementation of exec was not providing the errors. Repro-next was breaking, but the error was being swallowed. When switching to execa(and doing nothing more, so the error should happen), the error happened and the output was shown correctly

Reply:
Hmm, are you sure. I just tried again (on `next`):

<img width="755" alt="image" src="https://user-images.githubusercontent.com/132554/183012695-76f515cd-c07a-4967-bd02-b8901b257eea.png">

(it didn't do anything though)

Reply:
I didn't test by passing an outdir. I'll check

Reply:
Fixed! please try it out 👯